plasma/root_chain/contracts/RLPEncode.sol

```diff
@@ -0,0 +1,127 @@
+pragma solidity ^0.4.23;
+pragma experimental ABIEncoderV2;
+
+/// @title A simple RLP encoding library
```

Comment:
Can we switch the `///` style to the `/*` style used in the rest of our contracts?